.github/workflows/release-doc.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish release changelog
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build Changelog
+        id: pmm_release
+        run: |
+          git clone --depth 1 https://github.com/percona/pmm-doc doc
+          cd doc/docs/release-notes
+          version="${release_tag/refs\/tags\/v//}"
+          CHANGELOG=`cat ./$version.md`
+          echo "CHANGELOG<<EOF" >> $GITHUB_ENV
+          echo "$CHANGELOG" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```

Comment:
Not sure, but probably job output will fit better: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
It seems there is solution even better: https://github.com/softprops/action-gh-release#-external-release-notes

Reply:
::set-output does not work with multiline strings